docs/sample.tslint.json

```diff
@@ -96,7 +96,8 @@
     ],
     "triple-equals": [
       true,
-      "allow-null-check"
+      "allow-null-check",
+      "allow-undefined-check"
```

Comment:
Let's just keep "allow-null-check" in here for now - better to be using one or the other in